bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/GetMembershipsFlow.kt

```diff
@@ -81,10 +76,9 @@ class GetMembersFlow(private val forceRefresh : Boolean = false, private val fil
 
     override val progressTracker = tracker()
 
-    @Suspendable
-    override fun call(): List<PartyAndMembershipMetadata<Any>> {
+    override fun afterBNOIdentityVerified() : List<PartyAndMembershipMetadata<Any>> {
         progressTracker.currentStep = GOING_TO_CACHE_OR_BNO
-        return subFlow(GetMembershipsFlow(forceRefresh, filterOutNotExisting)).map { PartyAndMembershipMetadata(it.key, it.value.state.data.membershipMetadata) }
+        return subFlow(GetMembershipsFlow(bno, forceRefresh, filterOutNotExisting)).map { PartyAndMembershipMetadata(it.key, it.value.state.data.membershipMetadata) }
```

Comment:
I guess that if GetMembershipsFlow now returns all members (not just active) then this should filter out anyone who's not active.

Reply:
Good point. Changed.